src/services/loginService.js

```diff
@@ -0,0 +1,37 @@
+import Users from "../models/UserModel.js";
+
+export const createUser = async (email, password) => {
+	return await Users.create({
+		email: email,
+		password: password,
+	});
+};
+
+export const findAllUserByEmailID = async (email) => {
+	return await Users.findAll({
+		where: {
+			email: email,
+		},
+	});
+};
+
+export const findAllUserByRefreshToken = async (refreshToken) => {
+	return await Users.findAll({
+		where: {
+			refresh_token: refreshToken,
+		},
+	});
+};
```

Comment:
We don't need findall because refresh token is each user has unique token

Reply:
Fixed